devito/core/gpu_openmp.py

```diff
@@ -306,8 +393,11 @@ def _specialize_iet(cls, graph, **kwargs):
         data_manager.place_definitions(graph)
         data_manager.place_casts(graph)
 
-        return graph
+        # Initialize OpenMP environment
+        initialize(graph)
```

Comment:
should that name be changed?

Reply:
no, it's the same as in gpu_openacc, it's fine